src/test/test-models/bern_constrained_params_short.json

```diff
@@ -1 +1,4 @@
-{"params_r":[[0.5, -1.5, 2.6], [0.1, -3.5]]}
\ No newline at end of file
+{
```

Comment:
There were many boundary conditions in that code and tests that wind up throwing.  I don't see any of that getting tested.  Are there already tests in place for all this?  If not, I think they should be added if possible.

Reply:
There are some tests from the first PR in `src/test/interface/log_prob_test.cpp`. They didn't get updated in this PR besides to change the input data for them it seems